xwayland/xwayland.c

```diff
@@ -165,12 +165,9 @@ static void xwayland_finish_display(struct wlr_xwayland *wlr_xwayland) {
 
 	unlink_display_sockets(wlr_xwayland->display);
 	wlr_xwayland->display = -1;
-	unsetenv("DISPLAY");
+	wlr_xwayland->display_name[0] = '\0';
```

Comment:
rootston needs to unset this now

Reply:
This is not necessary. `xwayland_finish_display` is only called if `xwayland_start_display` fails and in `wlr_xwayland_destroy`. When this happens rootston will exit anyway.